packages/site/pages/components/avatar.js

```diff
@@ -36,6 +36,13 @@ export default _ => (
             null,
             'used to generate intials as a fallback image'
           ]),
+          PropTypes.row([
```

Comment:
What does having this (exposing this) solve for users?

Reply:
If you pass an avatar url that does not resolve to an avatar, and no name, the empty state is an empty circle. For my use case, a more sensible empty state would be to not render the avatar component at all. However, I don't know whether the Avatar will resolve the avatar url passed in. The solution I suggested was to expose a callback so that the client can know when the avatar does not resolve. Another option could be to take a render prop which Avatar calls to render the empty state. Yet another option would be to accept a boolean which indicates whether we want to render an `ident` gravatar in case the gravatar hash does not resolve to a real gravatar.

I put in a stop-gap solution to pass in a string that won't be undefined for the `name` prop for my use case, so this isn't critical for me at this time.